src/components/FileDetail/FileDetail.js

```diff
@@ -58,36 +57,18 @@ const FileDetail = ({fileNode, history, match}) => {
                       All Documents
                     </Button>
                   </Link>
-                </Grid.Column>
-              </Grid.Row>
-              <Grid.Row>
-                <Grid.Column mobile={16} tablet={16} computer={12}>
                   <Header as="h3" size="medium" color="blue">
                     {fileNode.name}
                   </Header>
                 </Grid.Column>
-                <Grid.Column mobile={16} tablet={16} computer={4}>
+                <Grid.Column mobile={2} tablet={2} computer={2}>
                   <Button.Group size="mini">
```

Comment:
If you add `attached="top"` and make the group `fluid` it will fix up some of the spacing mis-alignments.